### PR TITLE
Add more content to link outside website.

### PR DESCRIPTION
**Pull request by regulus33**
@mechamanny Hi Emmanuel! Can I get a code review?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
        </ul></aside>
     </form>
 
-    <p>For more info I recommend checking out <a href="http://www.livescience.com/52655-raccoons.html">Live Science's Raccoon Page.</a> They use a lot of the same html elements I used. Including an unordered list and a nav element. Their nav bar is neat because it moves with the user as you scroll through the page!
+    <p>For more info I recommend checking out <a href="http://www.livescience.com/52655-raccoons.html">Live Science's Raccoon Page.</a> They use a lot of the same html elements I used. Including an unordered list and a nav element. Their nav bar is neat because it moves with the user as you scroll through the page! I also noticed a script element I'd never seen before. The code inside the script element seemed to be javascript. I also noticed that the other added a lot of very helpful comments in their code.
```
Comment (mechamanny) [inline index.html]: Did you want a code review on this change?

Comment (regulus33) [inline index.html]: Hey! I was actually just requesting a code review on the solo challenge itself. In the 2.5 instructions it said "NOTE: If you do your work on a feature branch and create a PR against your master branch, you can tag your teacher on GitHub to request a code review of this challenge."

Comment (mechamanny) [inline index.html]: For future reference, I should be able to review the code from the PR screen, not go into your repo and search for the code to look at. 

The tagged branch should have all of the changes that you want me to review.

Comment (regulus33) [inline index.html]: Gotcha, thanks for letting me know.

**Pull request merged**